File: eTierMeeting_v2/SPA/src/app/_core/_models/eTM_Meeting_Log_Page.ts
export interface eTM_Meeting_Log_Page {
  record_ID: string;
  tU_ID: string;
  deptId: string;
  center_Level: string;
  tier_Level: string;
  class_Level: string;
  page_Name: string;
  start_Time: Date | string;
  end_Time: Date | null;
  click_Link: boolean | null;
  isViewFirst: boolean;
}

export interface eTM_Meeting_Log_PageParamDTO {
  record_ID: string;
  clickLinkKaizenSystem: boolean;
}

File: eTierMeeting_v2/API/Models/eTM_Meeting_Log_Page.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace eTierV2_API.Models
{
    public partial class eTM_Meeting_Log_Page
    {
        [Key]
        public Guid Record_ID { get; set; }
        [Required]
        [StringLength(10)]
        public string TU_ID { get; set; }
        [Required]
        [StringLength(20)]
        public string Center_Level { get; set; }
        [Required]
        [StringLength(3)]
        public string Tier_Level { get; set; }
        [Required]
        [StringLength(50)]
        public string Class_Level { get; set; }
        [Required]
        [StringLength(20)]
        public string Page_Name { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime Start_Time { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? End_Time { get; set; }
        public bool? Click_Link { get; set; }
    }
}
